index.html

```diff
@@ -51,20 +52,29 @@
       <h2>Introduction</h2>
 
 <p>W3C is a <strong>growing and global community</strong> where participants
-choose to work together, and it is committed to maintaining a positive working environment, where each participant feels appreciated and respected and where everyone adheres to the same high level of standards of personal behavior. In that process we experience differences in language, location, nationality, and experience. In such a diverse environment, 
-misunderstandings and disagreements happen, which in most cases can be resolved
-informally.</p>
+choose to work together. It is committed to maintaining a positive working environment,
+where each participant feels appreciated and respected and where everyone adheres to the same high level of standards of personal behavior. 
+In that process we experience differences in language, location, nationality, and experience. 
+In such a diverse environment, misunderstandings and disagreements happen, which in most cases can be resolved informally.
+Where that does not resolve the problem, this code outlines how W3C will act.</p>
 	    <ul>
 <li> W3C community participants should treat each other with respect and professionalism and be mindful of cultural differences.</li>
 <li>W3C community participants should communicate constructively and avoid insulting, unwelcome or demeaning behavior.</li>
 <li>W3C strictly prohibits discrimination, intimidation, harassment, and bullying of any kind and on any basis.</li>
-<li>W3C will not tolerate abusive behavior in any form, whether it is verbal, physical, sexual, or implied.</li>
+<li>W3C considers abusive behavior in any form unacceptable, whether it is verbal, physical, sexual, or implied.
+        If such behaviour occurs W3C will act to minimise the harm caused to participants and the likelihood of the behaviour being repeated.</li>
 		    </ul>
 
 <p>A <strong>Code of Ethics and Professional Conduct</strong> is useful to
 define accepted and acceptable behaviors and to promote high standards of
-professional practice. The goal of this code of conduct is to ensure transparency in moderation of the working group and to ensure that the working group is an environment where everyone can participate without fear of harassment.It also provides a benchmark for self evaluation and
-acts as a vehicle for better identity of the organization.</p>
+  professional practice. The primary goals of this code of conduct are twofold:</p>
+<ul>
+  <li>to ensure moderation of the working group is carried out with transparency and accountability, and</li>
```

Comment:
s/working group/group

also on the following line. The CEPC applies more broadly than working groups.

Reply:
@chaals please could you make this edit?

Reply:
done